src/StoreKeeper/WooCommerce/B2C/Imports/AbstractProductImport.php

```diff
@@ -232,4 +236,42 @@ protected function cleanDotPath(string $shop_product_path): string
     {
         return trim($shop_product_path, '.');
     }
+
+    protected function processItem($dotObject, array $options = [])
+    {
+        $ShopModule = $this->storekeeper_api->getModule('ShopModule');
+
+        $shopProductId = $dotObject->get('id');
+        try {
+            $woocommerceProductId = $this->doProcessProductItem($dotObject, $options);
+
+            if (false !== $woocommerceProductId) {
+                $ShopModule->setShopProductObjectSyncStatusForHook([
+                    'status' => self::SYNC_STATUS_SUCCESS,
+                    'shop_product_id' => $shopProductId,
+                    'extra' => [
+                        'product_id' => $woocommerceProductId,
+                        'view_url' => get_permalink($woocommerceProductId),
+                        'edit_url' => admin_url('post.php?post='.$woocommerceProductId).'&action=edit',
+                        'date_synchronized' => (new \DateTime())->format('Y-m-d H:i:s'),
+                        'plugin_version' => implode(', ', [
+                            StoreKeeperOptions::PLATFORM_NAME.': '.get_bloginfo('version'),
+                            StoreKeeperOptions::VENDOR.' plugin: '.STOREKEEPER_WOOCOMMERCE_B2C_VERSION,
+                        ]),
+                    ],
+                ]);
+            }
+
+            return $woocommerceProductId;
+        } catch (\Throwable $throwable) {
+            $ShopModule->setShopProductObjectSyncStatusForHook([
```

Comment:
I think you need to pass some data to

![image](https://user-images.githubusercontent.com/38119988/211510106-938ee774-20e5-43c3-a8c2-4c9b0e60b029.png)

in the message, just get some the message of the throwable

in details, you can pass some stack trace.

Reply:
Added throwable message and stack trace.

Reply:
Ok